README.md

```diff
@@ -1,13 +1,15 @@
 # Mongo API Project
 
-Replace this readme with your own information about your project. 
+This project is build with MongoDB where I fetched data from Mongo database using mongoose models.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+## Solution
 
-## The problem
+To do this project I first created my own data and converted it to JSON using https://www.csvjson.com/. The data I created consist of restaurants in Stockholm that has changing rooms for babys. In order to get all my data into my database I used the database seeding process.
```

Comment:
Cool! Where did you get the original data from?

Reply:
Thank you. I did this data with two friends on my maternity leave with purpose to create an app that was called Skötrumskartan and showed all restaurants in sthlm that had changing rooms for babys. A friend of us created the app that was really simple, but I have an idea  to do more with it and maybe add more functionality to it.